tests/foreman/ui_airgun/test_hostcollection.py

```diff
@@ -66,3 +243,262 @@ def test_positive_add_host(session):
         assert (
             hc_values['hosts']['resources']['assigned'][0]['Name'] == host.name
         )
+
+
+@tier3
+@upgrade
+def test_positive_install_package(
+        session, module_org, vm_content_hosts, vm_host_collection):
+    """Install a package to hosts inside host collection remotely
+
+    :id: eead8392-0ffc-4062-b045-5d0252670775
+
+    :expectedresults: Package was successfully installed on all the hosts
+        in host collection
+
+    :CaseLevel: System
+    """
+    with session:
+        session.organization.select(org_name=module_org.name)
```

Comment:
redundant line here and in all the rest tests - module org get's selected by default, there's no point in extra selection.